index.html

```diff
@@ -0,0 +1,136 @@
+<html lang="en">
+<head> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main-doc">
+         <header>
+            <h1>programming languages documentation</h1>
+            <nav id = "navbar">
+                <ul>
+                    <li> <a class = "nav-link" href="#introduction">introduction</a></li>
+                    <li><a class = "nav-link" href="#what_you_should_already_know">what you should already know</a></li>
+                    <li><a class = "nav-link" href="#javascript_and_java">javascript and java</a></li>
+                    <li><a class = "nav-link" href="#Hello_World">Hello World</a></li>
+                    <li> <a class = "nav-link" href="#variables">variables</a></li>
+                    <li><a class = "nav-link" href="#declaring_variables">declaring variables</a></li>
+                    <li><a class = "nav-link" href="#variable_scope">variable scope</a></li>
+                    <li><a class = "nav-link" href="#global_variables">global variables</a></li>
+                </ul>   
+        </nav>
+      </header>
+        <section id = "introduction"class = "main-section">
+            <header >
+             <h2>introduction</h2>
+            </header>
+            <ul>
+                <li>main1</li>
+                <li>main2</li>
+            </ul>
+            <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language.</p>
+            <p>Inside a host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to provide programmatic control over them. </p>
+
+        </section>
+        <section id = "what_you_should_already_know"class = "main-section">
+            <header >
+                <h1>what you should already know</h1>
+               </header>
+               <p>A general understanding of the Internet and the World Wide Web (WWW)</p>
+               <p>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript. </p>
+               <code>
+                let num = 5;
+                let num1 = 6;
+                let sum = num + num1;
+                </code> 
+                <ul>
+                    <li>already1</li>
+                    <li>already2</li>
+                </ul>
+        </section>
+        <section id = "javascript_and_java"class = "main-section">
+            <header >
+                <h1>javascript and java</h1>
+               </header>
```

Comment:
consider using  h1 tag for your main title only and lower titles for subtitles. this is to ensure that your page is SEO optimized

Reply:
- [ ] Good job making the highlighted review. 👏  However, I strongly suggest you use back strings for areas where you indicate tag, for example instead of h1, you put in `h1`. This gives a more clearcut view to the potential project owner. nice job 👍 
- [ ] Also, Emojis with a friendly reason of tune such as 👍 😃 😉 💯 will give a better appeal to you intending project owner